factorio-docker-watchdog.go

```diff
@@ -158,23 +160,23 @@ func updateVersion(buildinfo BuildInfo) {
 	defer os.RemoveAll(pathRepo)
 	logrus.Info("Cloned repo")
 
-	//err = gitCheckoutBranch(pathRepo, "update-"+version.String())
-	//if err != nil {
-	//	logrus.Panic(err)
-	//}
-	//logrus.Info("Checkout branch ", version)
+	err = gitCheckoutBranch(pathRepo, "update-buildinfo")
```

Comment:
If I read this correctly, we have the branch name hardcoded to `update-buildinfo` here. Is this intended?

I think proper behavior would be to directly push to `master`. Optimally this could be determined via env, such that we can set the branch dynamically

Reply:
this is just for testing purpose atm